scripts/steamcmd_installer.sh

```diff
@@ -2,7 +2,7 @@
 # steamcmd Base Installation Script
 #
 # Server Files: /mnt/server
-# Image to install with is 'ghcr.io/parkervcp/games:source'
+# Image to install with is 'ghcr.io/parkervcp/yolks:debian'
```

Comment:
The image to install would be `ghcr.io/parkervcp/installers:debian`. Yolks:debian is a runtime image
```suggestion
# Image to install with is 'ghcr.io/parkervcp/installers:debian'
```